### ISSUE TITLE
Issue with pip installation

### ISSUE THREAD

**Issue by arthursdp** [r0x0r/pywebview#273]
### Specification
- Platform: Windows 8
- Version: Pywebview 2.2 with Python 3.6

### Description

On windows the pip install of version 2.2 fails. (2.1 works fine). 

The log states that _(I anonymized it to avoid showing my folder hierarchy)_: 
 ```
Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\ANON\AppData\Local\Temp\pip-install-evzlp9cs\pyobjc-framework-Security\setup.py", line 22, in <module>
        extra_link_args=["-framework", "Security"]
      File "C:\ANON\AppData\Local\Temp\pip-install-evzlp9cs\pyobjc-framework-Security\pyobjc_setup.py", line 389, in Extension
        os_level = get_os_level()
      File "C:\ANON\AppData\Local\Temp\pip-install-evzlp9cs\pyobjc-framework-Security\pyobjc_setup.py", line 203, in get_os_level
        pl = plistlib.readPlist('/System/Library/CoreServices/SystemVersion.plist')
      File "c:\ANON\lib\plistlib.py", line 162, in readPlist
        with _maybe_open(pathOrFile, 'rb') as fp:
      File "c:\ANON\lib\contextlib.py", line 81, in __enter__
        return next(self.gen)
      File "c:\ANON\lib\plistlib.py", line 120, in _maybe_open
        with open(pathOrFile, mode) as fp:
    FileNotFoundError: [Errno 2] No such file or directory: '/System/Library/CoreServices/SystemVersion.plist'
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\ANON\AppData\Local\Temp\pip-install-evzlp9cs\pyobjc-framework-Security\
```

I don't know but i think that the problem is the installation of pyobjc on a Windows platform, i'm not sur this module supports this platform.

**Comment by shivaprsdv** [comment]
This should not happen. Pyobjc is not a required dependency on Windows. I
don't know why the script tries to install it. Roman made the change in #269,
he might be able to look into this.

**Comment by r0x0r** [comment]
Today I learnt that code setup.py is not executed at the install time, but rather at the build time. Python wheels use completely different syntax for conditional installs.
Anyhow I will fix this issue and put out a new release asap

**Comment by r0x0r** [comment]
It should be fixed now and 2.2.1 is out. 

As a consequence I had to give up the idea of choosing between QT and GTK. There is simply no way to deduce this information at the install time. As the result I reintroduced a [qt] install switch. GTK dependencies are better be installed via apt-get.
